print.go

```diff
@@ -177,6 +192,142 @@ var (
 
 func sprint(name string, node Node) string {
 	s := new(strings.Builder)
-	Print(s, name, node)
+	PrintSchema(s, name, node)
 	return s.String()
 }
+
+func PrintRowGroup(w io.Writer, rowGroup RowGroup) error {
+	schema := rowGroup.Schema()
+	pw := &printWriter{writer: w}
+	tw := tablewriter.NewWriter(pw)
+
+	mapping := ColumnMappingOf(schema)
+	columns := mapping.Columns()
+	header := make([]string, len(columns))
+	footer := make([]string, len(columns))
+	alignment := make([]int, len(columns))
+
+	for i, column := range columns {
+		_, node := mapping.Lookup(column...)
+		columnType := node.Type()
+
+		header[i] = strings.Join(column, ".")
+		footer[i] = columnType.String()
+
+		switch columnType.Kind() {
+		case ByteArray:
+			alignment[i] = tablewriter.ALIGN_LEFT
+		default:
+			alignment[i] = tablewriter.ALIGN_RIGHT
+		}
+	}
+
+	var row = make(Row, 0, rowGroup.NumColumns())
+	var err error
+
+	cells := make([]string, 0, len(columns))
+	parts := make([]string, 0)
+	rows := rowGroup.Rows()
+	for {
+		if row, err = rows.ReadRow(row[:0]); err != nil {
+			if !errors.Is(err, io.EOF) {
+				return err
+			}
+			break
+		}
+
+		cells = cells[:0]
+		fmt.Printf("%#v\n", row)
```

Comment:
Stray, I think?

Reply:
Yes, that was a miss, removed in https://github.com/segmentio/parquet-go/pull/146